docs-source/content/faq/external-clients.md

```diff
@@ -29,15 +29,15 @@ Here are the steps:
 
 - Review the [Security notes](#security-notes).
 
-##### Kubernetes NodePorts
+##### Kubernetes `NodePorts`
 
-The Kubernetes NodePorts approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes NodePort that redirects external network traffic on the Kubernetes nodes to the network channel.
+The Kubernetes `NodePorts` approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes `NodePort` that redirects external network traffic on the Kubernetes nodes to the network channel.
 
 Here are the steps:
 
 - Configure a custom channel for the T3 protocol in WebLogic that specifies an external address and port that are suitable for remote client use.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
 
-- Deploy a Kubernetes NodePort to publicly expose the WebLogic ports. See [Setting up a NodePort](#setting-up-a-nodeport).
+- Deploy a Kubernetes `NodePort` to publicly expose the WebLogic ports. See [Setting up a `NodePort`](#setting-up-a-nodeport).
```

Comment:
I don't think that "deploy" makes sense here, "create" is probably more accurate, or "define" or "expose"

Reply:
I like "expose".  Kubernetes documentation uses this term: https://kubernetes.io/docs/concepts/services-networking/service/

Reply:
I changed all the instances of "deploy" to "expose", except in this one bulleted item, where I thought that "define" was the correct term to use, "Define a Kubernetes `NodePort` to publicly expose the WebLogic ports. Please correct me if I'm wrong and I will change it to "expose."

Reply:
That sounds reasonable to me.